test/composers/requestMatchesComposer.test.js

```diff
@@ -122,6 +122,24 @@ describe('Network Request Match Composer', () => {
 			},
 		});
 
+		chain.requestMatch({
+			name: NETWORK_PROP.BODY,
```

Comment:
@AntonSBilan I think you should test METHOD here?

Reply:
fixed